src/typing_extensions.py

```diff
@@ -3818,6 +3824,7 @@ class Format(enum.IntEnum):
     VALUE = 1
     FORWARDREF = 2
     SOURCE = 3
+    STRING = 3
```

Comment:
I guess, `SOURCE` will be removed soon?

Reply:
We should just remove SOURCE since we never released this enum.